app/scripts/models/verification/same-browser.js

```diff
@@ -70,7 +70,17 @@ define(function (require, exports, module) {
     load () {
       var id = this._getUsersStorageId();
       if (id) {
-        var usersStoredInfo = (this._storage.get(STORAGE_KEY) || {})[id];
+        const allStoredInfo = this._storage.get(STORAGE_KEY) || {};
+        // password reset SameBrowserVerificationModel persist info
```

Comment:
The text read a little strange to me. Maybe

`During password reset, SameBrowserVerificationModel will persist info keyed by email since the uid is not known when the flow is initiated. During signup, it is keyed by uid. `